### DISCUSSION TITLE
Notes becoming stale

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2894] [General]
I have an issue where one of my notes will become stale - with the right pane either showing no text at all, or showing the text of the previously selected note.  Looking at the note history I can see that the data is still intact, and if I duplicate the note, the duplicate works just fine (until it eventually breaks again).

I don't have a consistent repro, but it happens randomly when selecting different notes.  Once the bug has occurred, the note is permanently bugged and has to be duplicated and deleted.  These are all plain text notes.  I'm using Windows 10.

**Comment by effieeee** [comment]
Hi @synthc, that's a rather strange issue, may I ask if you encounter this on the web app too?

**Comment by ghost** [comment]
> Hi @synthc, that's a rather strange issue, may I ask if you encounter this on the web app too?

I don't use the web app, but I tried selecting notes on there for a bit and didn't see it happen.  I do use the mobile app and it hasn't happened there either.

**Comment by effieeee** [comment]
Thanks for the added info, since this doesn't appear to happen on other app platforms so far, I wanted to ask if you can try either [clearing the site data](https://docs.standardnotes.com/troubleshooting/reset-apps/#desktop) for the desktop app? If this doesn't help either, may I ask you to try a clean reinstall of the app? (ensure that you follow the [uninstallation instructions](https://standardnotes.com/help/72/how-do-i-uninstall-standard-notes) before installing a fresh copy)

**Comment by ghost** [comment]
There is no "Clear Storage" button under the Application tab, but I cleared site data and that didn't fix the problem.  After that I reinstalled (following the linked instructions), but the problem still persists.

I've been leaving one bugged note around for testing and I noticed that the bug sort of moves between notes - that is to say one note will be bugged for a while and then it will start working again and a different note will stop working.

I've also noticed that the bug will eventually resolve itself and all notes will start working for a while.  No changes to my notes or app settings (or even an app restart) were made to cause this to happen.  All I'm doing is selecting different notes; this happens seemingly at random while the app is running.

**Comment by effieeee** [comment]
Hmm, that's truly bizarre 😢I'm afraid I'm not entirely certain what to try next until I can manage to replicate this, but please do keep me posted on any changes with the behavior. I hope the web app might be a suitable alternative for you in the meantime as well 😞🙏
